### PR TITLE
Team 2 procrastian create profile step 1

### PR DESCRIPTION
**Pull request by procrastian**

### CODE REVIEW

**Review by ManNavic — APPROVED** [review]
LGTM

**Review by JayVith — APPROVED** [review]
Looking Nice and seems to work which is always a good thing!

**Comment by dearshrewdwit** [comment]
[Create Profile - Step 1](https://trello.com/c/fuirgFqY/24-create-profile-step-1)

**Comment by dearshrewdwit** [comment]
Currently failing because: 

```
2023-05-23T09:40:45.168Z [INFO]: [eslint]
                                 src/components/form/textInput/index.js
                                 Syntax error: Unexpected token (3:1) (3:1)
```

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]

File: src/components/form/textInput/index.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import { useState } from "react";
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (procrastian) [reply src/components/form/textInput/index.js]: Just resolved these conflicts & updated

File: src/pages/welcome/stepOne/index.js [hunk 2]
```diff
@@ -1,38 +1,77 @@
...
+  };
+
+  const handleInput = (e) => {
+    setData(e);
+    checkValidFirstName(e);
+    checkValidLastName(e);
+  };
+
+  checkValidBothNames();
+  console.log(requiredFirstName, requiredLastName)
```
Comment (dearshrewdwit) [inline src/pages/welcome/stepOne/index.js:42]: remove log statement

**Review by dearshrewdwit — CHANGES_REQUESTED** [review]
Small change - remove log statement

File: src/pages/welcome/stepOne/index.js (hunk 2 above)
Comment (procrastian) [reply src/pages/welcome/stepOne/index.js]: Put it in to show the team how it was working... forgot to remove it again 🤦‍♂️ have removed it now

**Review by dearshrewdwit — APPROVED** [review]

**Review by Shylan21 — APPROVED** [review]
LGTM

**Pull request merged**